.gitignore

```diff
@@ -163,3 +163,6 @@ pkg/network/netlink/testdata/message_dump*
 # serverless
 .layers
 .extension
+
+# agent build script
+.ddagent
```

Comment:
sorry if I wasn't clear in my last comment :) since this is a personal script, we probably shouldn't make the repo aware of it. 

I think this doc covers it: https://stackoverflow.com/questions/7335420/global-git-ignore

Or you can just unstage and remove it from this PR.